src/Parser/CommandInfo.php

```diff
@@ -323,12 +323,9 @@ public function hasAnnotation($annotation)
      */
     public function addAnnotation($name, $content)
     {
-        // Convert to an array and merge if there are multiple
-        // instances of the same annotation defined.
-        if (isset($this->otherAnnotations[$name])) {
-            $content = array_merge((array) $this->otherAnnotations[$name], (array)$content);
-        }
-        $this->otherAnnotations[$name] = $content;
+        $values = $this->getAnnotationList($name);
+        $values[] = $content;
```

Comment:
`$values` will sometimes contain literal (non-list) values; these should not always be converted into a list.